firebase-database/src/test/java/com/google/firebase/database/util/ComparatorTester.java

```diff
@@ -165,11 +164,10 @@ private final void doTestEqualsCompatibility() {
       for (Object reference : referenceGroup) {
         for (List<Object> otherGroup : equalityGroups) {
           for (Object other : otherGroup) {
-            assertWithMessage(
-                    "Testing equals() for compatibility with compare()/compareTo(), "
-                        + "add a call to doNotRequireEqualsCompatibility() if this is not required")
+            // Testing equals() for compatibility with compare()/compareTo(),
+            // add a call to doNotRequireEqualsCompatibility() if this is not required
+            assertWithMessage("%s.equals(%s)", reference, other)
```

Comment:
Nit: Should the message have quotes around `%s`?

Reply:
LOL. you just couldn't let me have a clean review, eh?